jitpack.yml

```diff
@@ -0,0 +1,5 @@
+jdk:
+  - openjdk17
+install:
+   - echo "Publish aepsdk-gradle-plugin"
+   - ./android/aepsdk-gradle-plugin/gradlew -p ./android/aepsdk-gradle-plugin publishPluginJitpackPublicationToMavenLocal 
```

Comment:
Can we move this line to a makefile like the core implementation (https://github.com/adobe/aepsdk-core-android/blob/main/jitpack.yml)? I think it should be helpful to install the plugin locally for debugging or other purposes.